torch/ao/quantization/fx/prepare.py

```diff
@@ -774,10 +766,23 @@ def _recursive_maybe_replace_node_with_obs(
 
     new_args = []
     for old_arg in graph_output_node.args:
-        new_args.append(
-            _recursive_maybe_replace_node_with_obs(
-                old_arg, output_target_dtype, node_name_to_target_dtype,
-                qconfig_map, model, modules, graph))
+        if isinstance(old_arg, (list, tuple)):
+            for arg_idx, inner_node in enumerate(old_arg):
+                if arg_idx in output_quantized_idxs:
+                    output_target_dtype = output_quantized_idxs[arg_idx]
+                else:
+                    output_target_dtype = torch.quint8
```

Comment:
shouldn't this be `torch.float`?

Reply:
i think it should be quint8, see   L711 the original default output_target_dtype = torch.quint8

Reply:
L711 is set when we have output_quantized_idxs == [0] I think, in this branch the scenario is that there are multiple outputs, let's say 2 output, but user only specifies the dtype for one of them, e.g. {0: torch.quint8}, then for the one that user didn't mention, we should set the dtype to float I think. You should have a test case for this as well to catch this case I think

Reply:
I mean L711 in original code, it used quint8 by default.

Reply:
yeah I'm talking about original code as well, given the assert in L703 and return in L708, we are sure at that point `output_quantized_idxs == [0]`

Reply:
Got it. 
Dict support is already, check the code from L779